mutyper/cli.py

```diff
@@ -264,12 +266,14 @@ def iterate_with_ambiguity_warning():
         spectra_data = defaultdict(lambda: np.zeros_like(vcf.samples, dtype=int))
         if args.randomize:
             for variant in iterate_with_ambiguity_warning():
+                # counts = (variant.genotype.array()[:,:-1] == 1).sum(axis=1)
```

Comment:
I plan on pushing this line in a separate pull request because it drastically speeds up the computation, but I have deleted it for this one.

Reply:
hmm, can you just save it for that PR, or if it's really just one line like this, I think you could just uncomment it here.

Reply:
I'm cool with either approach, I think just to keep things separate (since the 2 commented lines were not written to address issue #36), maybe it's better to submit them as a separate PR. In my latest commit, I deleted the commented lines, but if you think it's not a big deal since it's just 2 lines, I can easily add them back again.